otmonitor.vfs/otmonitor.tcl

```diff
@@ -2136,6 +2139,12 @@ register 82	{f8.8}		"Exhaust inlet temperature"
 register 83	{f8.8}		"Exhaust outlet temperature"
 register 84	{u16}		"Exhaust fan speed"
 register 85	{u16}		"Inlet fan speed"
+register 98	{f8.8}		"For a specific RF sensor the RF strength and battery level is written" ;# Data-type assumed
+register 99	{f8.8}		"Operating Mode HC1, HC2/ Operating Mode DHW"				;# Data-type assumed
```

Comment:
These messages don't have data type f8.8. Instead they use bit fields. So they may need special treatment, similar to the date and time messages (20 & 21).